scout/commands/load/variants.py

```diff
@@ -42,11 +41,17 @@
     help="Specify the rank score treshold",
     show_default=True,
 )
+@click.option("-f", "--force", is_flag=True, help="upload without request")
+@click.option(
+    "-k",
+    "--keep-tags",
```

Comment:
I think this should be the default behaviour!
```suggestion
    "--keep-tags/--no-keep-tags', default=True,",
```

Reply:
Or make the flag `--kill-old-user-tags`.. 😸

Reply:
> I think this should be the default behaviour!

I think that most users would agree. I'll make it default!